api/src/main/java/org/openmrs/module/esaudemetadata/metadatadeploy/CommonMetadataBundle.java

```diff
@@ -11,90 +11,134 @@
+		install(encounterType("S.TARV: BUSCA ACTIVA", "Busca Activa",
+				MetadataBundleUtils._EncounterType.S_TARV_BUSCA_ACTIVA));
+		install(encounterType("S.TARV: FARMACIA", "Farmacia", MetadataBundleUtils._EncounterType.S_TARV_FARMACIA));
+		install(encounterType("S.TARV: LIVRO PRE-TARV", "Representa o livro de registo PRE-TARV",
+				MetadataBundleUtils._EncounterType.S_TARV_LIVRO_PRE_TARV));
+		install(encounterType("S.TARV: LIVRO TARV", "Representa o livro de registo TARV",
+				MetadataBundleUtils._EncounterType.S_TARV_LIVRO_TARV));
+		install(encounterType("S.TARV: PEDIATRIA INICIAL A", "Representa o livro de registo TARV",
+				MetadataBundleUtils._EncounterType.S_TARV_PEDIATRIA_INICIAL_A));
+		install(encounterType("S.TARV: PEDIATRIA INICIAL B",
+				"Primeira visita de paciente pediatrico, parte B. As duas ultimas paginas",
+				MetadataBundleUtils._EncounterType.S_TARV_PEDIATRIA_INICIAL_B));
+		install(encounterType("S.TARV: PEDIATRIA SEGUIMENTO", "Seguimento visita do paciente pediatria",
+				MetadataBundleUtils._EncounterType.S_TARV_PEDIATRIA_SEGUIMENTO));
+		install(encounterType("S.TARV: SOLICITAÇÃO ARV", "Solicitação de Medicamentos Anti-Retrovirais",
+				MetadataBundleUtils._EncounterType.S_TARV_SOLICITACAO_ARV));
+		install(encounterType("S.TARV: TERMO DE CONSENTIMENTO DE VISITA",
+				"Termo de consentimento de visita domiciliaria/busca activa",
+				MetadataBundleUtils._EncounterType.S_TARV_TERMO_DE_CONSENTIMENTO_DE_VISITA));
+		install(encounterType("TRANSFER", "TRANSFER encounter", MetadataBundleUtils._EncounterType.TRANSFER));
+		install(encounterType("TUBERCULOSE: LIVRO", "Livro de Registo de Doentes com Tuberculose",
+				MetadataBundleUtils._EncounterType.TUBERCULOSE_LIVRO));
+		install(encounterType("TUBERCULOSE: PROCESSO", "TRATAMENTO DE TUBERCULOSE",
+				MetadataBundleUtils._EncounterType.TUBERCULOSE_PROCESSO));
+		install(encounterType("TUBERCULOSE: RASTREIO", "TUBURCULOSE: RASTREIO",
+				MetadataBundleUtils._EncounterType.TUBERCULOSE_RASTREIO));
+		install(encounterType("TUBERCULOSE: SEGUIMENTO", "Seguimento de doentes com tuberculose",
+				MetadataBundleUtils._EncounterType.TUBERCULOSE_SEGUIMENTO));
 
+		// Install forms
+		install(form("ADULTO: PROCESSO PARTE A - ANAMNESE", "Processo de Hospital de Dia 1/2",
+				MetadataBundleUtils._EncounterType.S_TARV_ADULTO_INICIAL_A, "1.1",
+				MetadataBundleUtils._Form.ADULTO_PROCESSO_PARTE_A_ANAMNESE));
+		// install Location attribute
 
-        //Install forms
-        install(form("ADULTO: PROCESSO PARTE A - ANAMNESE", "Processo de Hospital de Dia 1/2", MetadataBundleUtils._EncounterType.S_TARV_ADULTO_INICIAL_A, "1.1", MetadataBundleUtils._Form.ADULTO_PROCESSO_PARTE_A_ANAMNESE));
-        //install Location attribute
+		// Install Encounter Role
+		// global properties
 
-        // Install Encounter Role
-        //global properties
+		// installing privileges
+		install(privilege(MetadataBundleUtils._Privilege.READ_TEST_ORDER, "View Test Orders"));
```

Comment:
I think we should be using ```Read``` instead of ```View``` for this kind of privilege according to [docs](https://docs.google.com/spreadsheets/d/1HJioc0wpZUsjjIkVM3t_vpb5UJkTtJ1V1i-VOdA_Px0/edit#gid=247616070).